bin/oref0-setup.sh

```diff
@@ -1197,7 +1227,7 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
         (crontab -l; crontab -l | grep -q "oref0-bluetoothup" || echo '* * * * * ps aux | grep -v grep | grep bash | grep -q "oref0-bluetoothup" || oref0-bluetoothup >> /var/log/openaps/network.log' ) | crontab -
         fi
         # proper shutdown once the EdisonVoltage very low (< 3050mV; 2950 is dead)
-        if egrep -i "edison" /etc/passwd 2>/dev/null; then
+        if egrep -i "edison" /etc/passwd &>/dev/null; then
```

Comment:
Why change this?

Reply:
It's dumping it to stdout during the run of the setup, not needed... I originally noticed /etc/passwd coming up a bunch through the run which made me wonder why